pkgs/development/ocaml-modules/h2/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildDunePackage
+, fetchFromGitHub
+, hpack
+, angstrom
+, faraday
+, base64
+, psq
+, httpaf
+, alcotest
+, yojson
+, hex
+}:
+
+let
+  http2-frame-test-case = fetchFromGitHub {
```

Comment:
why prefer this instead of pulling the submodules?

Reply:
I wanted to use the release tar ball if possible which mainly motivated this. If you don't actually rely on `dune subst` or the likes we can probably use `fetchSubmodules = true`.